.vib/kibana/goss/vars.yaml

```diff
@@ -0,0 +1,22 @@
+binaries:
+  - kibana
+  - kibana-keystore
+  - yq
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/kibana
+      - /bitnami/kibana/data
+      - /opt/bitnami/kibana/config
+      - /opt/bitnami/kibana/logs
+      - /opt/bitnami/kibana/plugins
+      - /opt/bitnami/kibana/tmp
+      - /docker-entrypoint-initdb.d
+files:
+  - paths:
+      - /opt/bitnami/kibana/config/kibana.yml
```

Comment:
You are already checking this file on `kibana.yaml` so the test here will be unnecessary. More so, having 2 tests checking the same `file` (or 2 `command` tests with the same name) is not supported by Goss; it will only execute one of them, overwriting the other.

Reply:
Fixed!